lib/config/presets/internal/monorepo.ts

```diff
@@ -263,6 +263,7 @@ const orgGroups = {
   ngrx: 'https://github.com/ngrx/',
   nrwl: 'https://github.com/nrwl/',
   'semantic-release': 'https://github.com/semantic-release/',
+  swc: 'https://github.com/swc-project',
```

Comment:
```suggestion
  swc: 'https://github.com/swc-project/',
```
so `swc-project1` isn't matched 😏